### PR TITLE
Add more emotions and hug types.

### PR DESCRIPTION
**Pull request by JamesHawkinss**
I got bored and found your GitHub :D

### CODE REVIEW

File: src/main/java/uk/co/binaryoverload/hugapi/Hug.java [hunk 1]
```diff
@@ -106,6 +106,7 @@ public Hug create() {
         MUM_HUG,
         COMFORTING,
         BRITISH,
+        PHATT,
```
Comment (binaryoverload) [inline src/main/java/uk/co/binaryoverload/hugapi/Hug.java:109]: What in mother of mary is this... xD

Comment (JamesHawkinss) [reply src/main/java/uk/co/binaryoverload/hugapi/Hug.java:109]: A really squishy warm hug, slang in my secondary school is ott

Comment (binaryoverload) [reply src/main/java/uk/co/binaryoverload/hugapi/Hug.java:109]: Find something else that people can understand 👀

Comment (JamesHawkinss) [reply src/main/java/uk/co/binaryoverload/hugapi/Hug.java:109]: okey dokey

**Pull request closed without merging**